MH-FiorellaTheme5/source/stylesheets/styles.css

```diff
@@ -659,9 +730,34 @@ p{
     text-align: center;
   }
 
+  .title-icon{
+    top: 40px;
+  }
+
+  .promotion-message{
+    left: 40px;
+    bottom: 40px;
+    font-size: 40px;
+  }
+
+  .box-promotions{
```

Comment:
1. missing space before `{`
2. Please reorder the properties in alphabetical order